index.html

```diff
@@ -1,33 +1,209 @@
+        <nav>
+            <i class="material-symbols-outlined menu_icon">
+                menu
+            </i>
+        </nav>
     </header>
     <section class="container">
         <h1>I’m Abdurahim Glad to see you!</h1>
-        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
         <h2 class="social_media">LET’S CONNECT</h2>
         <ul class="social_media_icons">
             <li><a href="#"><img src="images/icons/Twitter.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Media.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Github.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Sideshow.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/icons/Media.svg" alt="media"></a></li>
+            <li><a href="#"><img src="images/icons/Github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images/icons/Sideshow.svg" alt="slideshow"></a></li>
         </ul>
     </section>
+    <section class="works_container">
+        <div class="card">
+            <img src="images/Snapshoot Portfolio.png" alt="">
+            <div class="work_desc">
+                <h1>Tonic</h1>
+                <div class="heading_3">
+                    <h3 class="main_h3">CANOPY</h3>
+                    <span></span>
+                    <h3>Back End Dev</h3>
+                    <span></span>
+                    <h3>2015</h3>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="action">
+                    <button class="btn">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="card">
+            <img src="images/Snapshoot Portfolio 2.png" alt="">
+            <div class="work_desc">
+                <h1>Multi-Post <br> Stories</h1>
+                <div class="heading_3">
+                    <h3 class="main_h3">CANOPY</h3>
+                    <span></span>
+                    <h3>Back End Dev</h3>
```

Comment:
- [ ] Kindly use an indentation of two spaces in all your documents. This is a convention and also best practice,